.travis.yml

```diff
@@ -22,7 +22,6 @@ matrix:
 
 install:
   - pip install -U tox-travis coveralls pip setuptools wheel
-  - python setup.py build_ext --inplace
```

Comment:
This isn't actually needed, building will happen via tox.

Reply:
Agreed. Think this was causing the build to happen twice before actually.